Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/opensaml/CertificateManagerImpl.java

```diff
@@ -57,10 +57,10 @@
     private KeyStore trustStore = null;
 
     public static final String TRUST_STORE_TYPE_KEY = "javax.net.ssl.trustStoreType";
-    public static final String TRUST_STORE_PASSWORD_KEY = "javax.net.ssl.trustStorePassword";
+    public static final String TRUST_STORE_SECRET_KEY = "javax.net.ssl.trustStorePassword";
```

Comment:
Out of curiosity, why was this renamed?  "Trust Store Password Key" isn't perfect, but it's more descriptive and accurate than the new name -- the String is the key in the key/value pair for the trust store password.